lib/spack/spack/build_environment.py

```diff
@@ -240,6 +244,7 @@ def set_module_variables_for_package(pkg, module):
     # TODO: make these build deps that can be installed if not found.
     m.make  = MakeExecutable('make', jobs)
     m.gmake = MakeExecutable('gmake', jobs)
+    m.scons = MakeExecutable('scons', jobs)
```

Comment:
I'm just wondering if a package doesn't depend on scons and you try to build it that package, will build_environment.py get called and will MakeExecutable complain that it can't find scons?